crates/sui-source-validation-service/config.toml

```diff
@@ -36,3 +36,23 @@ packages = [
     { path = "crates/sui-framework/packages/sui-framework", watch = "0x2" },
     { path = "crates/sui-framework/packages/sui-system", watch = "0x3" },
 ]
+
+[[packages]]
+source = "Repository"
+[packages.values]
+repository = "https://github.com/mystenlabs/apps"
+branch = "main"
+network = "mainnet"
+packages = [
+    { path = "kiosk" },
+]
```

Comment:
This doesn't add an object to `watch` for upgrades (if @damirka knows the `UpgradeCap` or similar wrapped object to watch we can add it here now or in future). We don't strictly need it though: The significance of watching a third-party upgrade is that we will immediately be notified / kick the server so that we then immediately pick up the latest source changes in the branch (and presumably, find and track the new `published-at` address).

Said another way: If we don't add a `watch`ed object, it simply means we won't explicitly get notified / kick the server when or if the package is upgraded, and we will continue serving the "old" source at the "old" address. Once the server restarts (for any other reason), we will pick up whatever the latest published package is in `Move.toml`.

Reply:
When do you need an UpgradeCap? I can look for it this week.

Reply:
Not urgent, we can add it any time later too, just giving context for all reviewers :-) We won't be using it until if/when Kiosk is upgraded again (and even then we don't strictly need it)